monocle/worker.py

```diff
@@ -831,7 +831,52 @@ def travel_speed(self, point):
                         pokestop = self.normalize_pokestop(fort)
                         db_proc.add(pokestop)
                 elif fort not in FORT_CACHE:
-                    db_proc.add(self.normalize_gym(fort))
+                    request = self.api.create_request()
+                    request.gym_get_info(
+                                            gym_id=fort.id,
+                                            player_lat_degrees = self.location[0],
+                                            player_lng_degrees = self.location[1],
+                                            gym_lat_degrees=fort.latitude,
+                                            gym_lng_degrees=fort.longitude
+                                        )
```

Comment:
this is indeed the only reliable way I found for it a.t.m.
the pogodev API page isn't up to date, otherwise that could be used.